lib/screens/search/explore_category_sort.dart

```diff
@@ -0,0 +1,92 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import '../../components/option_list.dart';
+
+class ExploreCategorySort extends StatefulWidget {
+  @override
+  State<ExploreCategorySort> createState() => _ExploreCategorySortState();
+}
+
+class _ExploreCategorySortState extends State<ExploreCategorySort> {
+  final optionData = [
+    Option(id: 'recently_published', title: 'Recently published'),
+    Option(id: 'a-z', title: 'A - Z'),
+  ];
+
+  String selectedOption = 'recently_published';
+
+  void onSelectionChanged(String id) {
+    setState(() {
+      selectedOption = id;
+    });
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return ListView(
+      padding: const EdgeInsets.symmetric(horizontal: 16, vertical: 8),
+      physics: const NeverScrollableScrollPhysics(),
```

Comment:
You are right 👍. When I tried column initially, it was taking full height. So I used ListView with shrinkWrap property so that it occupies minimum height.
Now I found the minAxisSize property for Column which achieves the same thing. So I have now changed it to Column.